docs/book/src/user/quick-start.md

```diff
@@ -1357,18 +1358,42 @@ Note: To use the default clusterctl method to retrieve kubeconfig for a workload
 {{#/tab }}
 {{#/tabs }}
 
+### Install a Cloud Provider
+
+The Kubernetes in-tree cloud provider implementations are being [removed](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cloud-provider/2395-removing-in-tree-cloud-providers) in favor of external cloud providers (i.e. out-of-tree). To learn more, see [this blog post](https://kubernetes.io/blog/2019/04/17/the-future-of-cloud-providers-in-kubernetes/).
+
+{{#tabs name:"tab-install-cloud-provider" tabs:"Azure"}}
+{{#tab Azure}}
+
+Grab the CIDR ranges from your cluster by running this kubectl statement against the management cluster:
+
+```bash
+export CCM_CIDR_BLOCK=$(kubectl get cluster "${CLUSTER_NAME}" -o=jsonpath='{.spec.clusterNetwork.pods.cidrBlocks[0]}')
```

Comment:
If we assume they used the quickstart, can we provide a default as such so they can potentially skip this step?

Reply:
the cloud-provider-azure Helm chart default is different from the CAPZ cluster default so, unfortunately, no

Reply:
but I could hardcode it in the Helm install command if that's what you mean. Is that better?

Reply:
Did the latter, ptal